services/vision/server.go

```diff
@@ -217,6 +200,54 @@ func segmentsToProto(frame string, segs []*vision.Object) ([]*commonpb.PointClou
 	return protoSegs, nil
 }
 
+func (server *serviceServer) CaptureAllFromCamera(
+	ctx context.Context,
+	req *pb.CaptureAllFromCameraRequest,
+) (*pb.CaptureAllFromCameraResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "service::vision::server::CaptureAllFromCamera")
+	defer span.End()
+	svc, err := server.coll.Resource(req.Name)
+	if err != nil {
+		return nil, err
+	}
+	capt, err := svc.CaptureAllFromCamera(ctx,
+		req.CameraName,
+		req.ReturnImage,
+		req.ReturnDetections,
+		req.ReturnClassifications,
+		req.ReturnObjectPointClouds,
+		req.Extra.AsMap(),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	objProto, err := segmentsToProto(req.CameraName, capt.PointCloudObject())
```

Comment:
The name of the variable from the API is "Objects()"  -- this is why you need to write tests